doc/preview_install_guide.md

```diff
@@ -196,7 +196,7 @@ Once the install is complete, if you open a new command prompt, you should test
 
 #### Step 2: Install Azure CLI using PIP
 
-Run the following from a command prompt to install the Azure CLI 2.0 using the Python package manager, PIP:
+Run the following from an administrator command prompt to install the Azure CLI 2.0 using the Python package manager, PIP:
```

Comment:
The user doesn't have to be an administrator to install with pip.
Though if you simply do `pip install azure-cli` you likely will have to be as it attempts to install globally.

There are a couple of options if you are not an administrator:
1. User a virtual environment and install the CLI in there.
```
> virtualenv env
> env \Scripts\activate
> pip install azure-cli
```

2. Use pip to install in user directory
```
> pip install --user azure-cli
```